src/user/components/navbar/components/UserMenu/UserMenu.tsx

```diff
@@ -1,19 +1,32 @@
 import * as React from "react";
 
 import { useHistory } from "react-router-dom";
-import { Badge, Button, Menu, MenuItem } from "@mui/material";
+import {
+  Button,
+  CircularProgress,
+  Menu,
+  MenuItem,
+  Snackbar,
+} from "@mui/material";
 import { useState } from "react";
-import DraftsIcon from "@mui/icons-material/Drafts";
+import { useSignOut } from "../../../../../auth/hooks/mutations/useSignOut";
```

Comment:
Can we set `"baseUrl": "src"` in tsconfig and use import paths from the root like we do in ads-admin?  All those dot dots make my eyes go funny. (Probably makes sense for that to be in a subsequent PR rather than this one though.)

Reply:
Created: https://github.com/brave/ads-ui/issues/735